org.spoofax.terms/src/org/spoofax/terms/io/TextTermWriter.java

```diff
@@ -0,0 +1,399 @@
+package org.spoofax.terms.io;
+
+import org.spoofax.interpreter.terms.*;
+import org.spoofax.terms.attachments.ITermAttachment;
+
+import java.io.IOException;
+
+/**
+ * Default term writer.
+ *
+ * Override member of this class to influence term writing.
+ * Overriding implementation should make sure to call {@link #writeTerm} to write subterms,
```

Comment:
This can be in an `@implSpec` section.

Reply:
I ddin't know about `@implSpec` and the other Java 8 JavaDoc tags.